### PR TITLE
vstudio staticlib symbolspath

### PR DESCRIPTION
**Pull request by tdesveauxPKFX**
ref: #1032 

Also fixed names from tests I added in #1030

### CODE REVIEW

File: modules/vstudio/vs2010_vcxproj.lua [hunk 1]
```diff
@@ -2175,14 +2181,15 @@
 		end
 	end
 
-
 	function m.programDatabaseFile(cfg)
 		if cfg.symbolspath and cfg.symbols ~= p.OFF and cfg.debugformat ~= "c7" then
-			m.element("ProgramDatabaseFile", nil, p.project.getrelative(cfg.project, cfg.symbolspath))
+
+			local key = iif(cfg.kind == p.STATICLIB, "ProgramDataBaseFileName", "ProgramDatabaseFile")
```
Comment (tvandijck) [inline modules/vstudio/vs2010_vcxproj.lua]: Personally, I would prefer this to be split into two methods, one for "ProgrameDatabaseFileName", and one for "ProgramDatabaseFile"..

While I get that they will be almost identical except for their name, it allows for easier override functionality...

**Comment by tvandijck** [comment]
Thank you for all the added tests, much appreciated.

File: modules/vstudio/vs2010_vcxproj.lua (hunk 1 above)
Comment (tdesveauxPKFX) [reply modules/vstudio/vs2010_vcxproj.lua]: That was my first implementation but that would duplicate the condition and would be a pain to maintain IMO.
But I get your point about override, if you feel that's the way to go, I have no objection.
I will make the changes later today.

**Comment by tdesveauxPKFX** [comment]
This solution sounds good to me, I will implement it later today when I have some time.

**Pull request merged**